distribution/sender/routes/sender-message-consumer.xml

```diff
@@ -0,0 +1,80 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+    <route id="sender-message-consumer" errorHandlerRef="shutdownErrorHandler">
```

Comment:
How about naming this sender-activemq-consumer or sender-activemq-message-consumer

Reply:
Renamed to sender-activemq-consumer.